react-exercise/src/components/tabs/Tabs.scss

```diff
@@ -0,0 +1,40 @@
+.tabs {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+  }
+  
+  .tab-buttons {
+    display: flex;
+    gap: 16px; // Space between the tabs
+    margin-bottom: 16px;
+    border-bottom: 1px solid var(--semantic-bg-neutral-inverse, #E6E7E9);
+  }
+  
+  .tab-button {
+    background: none;
+    border: none;
+    min-width: 72px;
+    padding: 16px 0px;
+    cursor: pointer;
+
+    color: var(--semantic-fg-secondary, #7A7C81);
+    font-family: "Brown LL";
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    letter-spacing: -0.5px;
+
+    border-radius: var(--borderRadius, 4px) var(--borderRadius, 4px) 0px 0px;
+  }
+  
+  .tab-button.active {
```

Comment:
We usually put this inside the ".tab-button {" definition. 
Your way still works, just a convention + a bit more organized IMO.